tfjs-backend-webgpu/README.md

```diff
@@ -1,13 +1,90 @@
-This is an experimental backend.
+# Usage
 
+This package adds a GPU accelerated [WebGPU](https://www.w3.org/TR/webgpu/) backend to TensorFlow.js. It currently supports
+the following models:
+- BlazeFace
+- BodyPix
+- Face landmarks detection
+- HandPose
+- MobileNet
+- PoseDetection
+- Universal sentence encoder
+- AutoML Image classification
+- AutoML Object detection
+- Speech commands
+
+Note that WebGPU 1.0 hasn't been officially released by W3C group. Currently WebGPU in chrome is in Origin Trial. To try webgpu features, you need to launch chrome canary browser with flag --enable-unsafe-webgpu.
+
+
+## Importing the backend
+
+### Via NPM
+
+```js
+// Import @tensorflow/tfjs or @tensorflow/tfjs-core
+import * as tf from '@tensorflow/tfjs';
+// Adds the WebGPU backend to the global backend registry.
+import '@tensorflow/tfjs-backend-webgpu';
+// Set the backend to WebGPU and wait for the module to be ready.
+tf.setBackend('webgpu').then(() => main());
 ```
-    $ yarn # to install dependencies
-    $ yarn test
+
+### Via a script tag
+
+```html
+<!-- Import @tensorflow/tfjs or @tensorflow/tfjs-core -->
+<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs/dist/tf.min.js"> </script>
+
+<!-- Adds the WebGPU backend to the global backend registry -->
+<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-webgpu/dist/tf-backend-webgpu.js"></script>
+<script>
+tf.setBackend('webgpu').then(() => main());
+</script>
+```
+
+# FAQ
+
+### When should I use the WebGPU backend?
+WebGPU as the successor of WebGL will provide a hardware accelerated solution.
+Theoretically, webgpu is recommended to be used in any scenario used by WebGL
+and provide comparable or better performance.
+
+We are committed to supporting the WebGPU backend and will continue to improve
+performance. We plan to follow the WebGPU standard closely and benefit from
+its upcoming features such as fp16, int8.
+
```

Comment:
How about this: "The mission of WebGPU backend is to achieve the best performance among all approaches. However, this target can not be met overnight, but we are committed to supporting it with rapid and continuous performance improvement. Many exciting features, like FP16, DP4A, will be brought in soon."

Reply:
Done